src/app.ts

```
@@ -136,13 +142,14 @@ app.get("/stream", (c: Context) => {
     start(controller) {
       controller.enqueue(
-        `data: ${JSON.stringify({ type: "connected", connectionId })}\n\n`
+        `data: ${JSON.stringify({ type: "connected", connectionId })}\n\n`,
       );
 
       const onMessage = (_channel: string, message: string) => {
         try {
           controller.enqueue(`data: ${message}\n\n`);
-        } catch (err) {
+        }
+        catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(unused-imports/no-unused-vars)